repos/system_upgrade/common/actors/inhibitwhenluks/actor.py

```diff
@@ -13,37 +13,9 @@ class InhibitWhenLuks(Actor):
     """
 
     name = 'check_luks_and_inhibit'
-    consumes = (StorageInfo, CephInfo)
-    produces = (Report,)
+    consumes = (CephInfo, LuksDumps, StorageInfo)
+    produces = (Report, TargetUserSpaceUpgradeTasks)
```

Comment:
Missing ``UpgradeInitramfsTasks``

Reply:
fixed. as you found it worked even without this msg produced, probably it's just seatbelt and dracut modules are automatically included when rpms are installed inside the container. but let's keep the seatbelt and make it active :-D fixed